File: Célia_LingProgram/ListaJavaFacul/src/L3_Exer10_Login/Exer10_Login.java
package L3_Exer10_Login;

import java.util.Scanner;

public class Exer10_Login {
// Elaborar um programa java que simule a autenticação de um
// usuário em um sistema. Solicite o nome de usuário e senha.
// Enquanto o nome for diferente de “admin” e a senha for diferente
// de “#Bandtec” exiba: “Login e/ou senha inválidos” e solicite novamente.
// Caso o usuário digite os campos corretamente, exiba: “Login realizado
// com sucesso
    public static void main(String[] args) {
        Scanner leitor = new Scanner(System.in);
        System.out.println("Insira seu login");
        String login = leitor.nextLine();
        System.out.println("Insira sua senha");
        String senha = leitor.nextLine();

        if(login.equals("admin") && senha.equals("#Bandtec")){
            System.out.println("Login realizado com sucesso!");
        }else{
            System.out.println("Login e /ou senha inválidos");
        }

    }
}

File: Célia_LingProgram/JavaFacul/src/Lista03/Exer10_Login.kt
package Lista03

import java.util.*

//
//Elaborar um programa java que simule a autenticação de um usuário em um sistema. Solicite o nome de usuário e senha.
// Enquanto o nome for diferente de “admin” e a senha for diferente de “#Bandtec” exiba: “Login e/ou senha inválidos” e
// solicite novamente.
// Caso o usuário digite os campos corretamente, exiba: “Login realizado com sucesso”

fun main() {
    val scanner = Scanner(System.`in`)

    println("Login")
    println("Insira seu usuário: ")
    val usuario = scanner.nextLine()
    println("Insira sua senha: ")
    val senha = scanner.nextLine()

    if(usuario.equals("admin") && senha.equals("#Bandtec")){
        println("Login realizado com sucesso!")
    }else{
        println("Login e/ou senha inválidos.")
    }
}